sdk/src/main/java/io/dapr/client/DaprHttpBuilder.java

```diff
@@ -33,6 +36,13 @@
    */
   private static final Object LOCK = new Object();
 
+  /**
+   * HTTP keep alive duration in seconds.
+   *
+   * <p>Just hard code to a reasonable value.
+   */
+  private static final int KEEP_ALIVE_DURATION = 30;
```

Comment:
Make this also a configurable parameter?

Reply:
I think this value can be safe to hard code. 

I have to set this because it's required by the constructor:

ConnectionPool pool = new ConnectionPool(Properties.HTTP_CLIENT_MAX_IDLE_CONNECTIONS.get(),
                  KEEP_ALIVE_DURATION, TimeUnit.SECONDS);

Reply:
We will remove OKHTTP dependency, so it is OK to hardcode this one for now.